src/psyclone/tests/utils.py

```diff
@@ -28,3 +50,141 @@ def count_lines(root, string_name):
         if string_name in line:
             count += 1
     return count
+
+
+def walk(parent, my_type):
```

Comment:
I think this can be removed if the changes proposed below are possible (see next comment).

Reply:
Removed.